### PR TITLE
Use Azure Storage static web serving rather than public access

### PR DESCRIPTION
**Pull request by matthchr**
**If applicable**:
- [ ] this PR contains documentation
- [ ] this PR contains tests
- [ ] this PR contains YAML Samples

### CODE REVIEW

File: scripts/v2/create-kind-wi-storage.sh [hunk 1]
```diff
@@ -42,24 +44,26 @@ if [ -f "$DIR/azure/rg.txt" ]; then
   exit 0
 fi
 
+ISSUER_URL="https://${AZURE_STORAGE_ACCOUNT}.z22.web.core.windows.net/"
```
Comment (nojnhuh) [inline scripts/v2/create-kind-wi-storage.sh:47]: Is this `z22` deterministic? When I just tried out these steps manually mine happens to be `z13`.

Comment (matthchr) [reply scripts/v2/create-kind-wi-storage.sh:47]: Thats a good question... it may not be fixed let me see if I can get it dynamically

Comment (nojnhuh) [reply scripts/v2/create-kind-wi-storage.sh:47]: Here's a command that shows me my URL if that's helpful:
```
az storage account show --name "${AZURE_STORAGE_ACCOUNT}" -o json | jq -r .primaryEndpoints.web
```

Comment (matthchr) [reply scripts/v2/create-kind-wi-storage.sh]: Fixed this, thanks!

**Review by super-harsh — APPROVED** [review]

**Pull request merged**